website/views/pages/homepage.ejs

```diff
@@ -51,8 +51,8 @@
     <div style="max-width: 800px" class="pb-5 mx-auto">
       <div style="max-width: 800px; text-align: center;" class="position-relative mx-auto">
         <p>"Fleet is hands down the best osquery platform out there. Zach and I created Fleet as a natural extension to our original vision for osquery and this has manifested in the ease-of-use, flexibility, and adoption it has today. Look to Fleet for the future of open-source endpoint monitoring."</p>
-        <p style="color: #676A7A" class="mb-4">- Mike Arpaia, Co-creator of osquery</p>
-        <img alt="osquery logo" src="/images/logo-osquery-89x24@2x.png" style="height: 40px; width: 137px;" class="mb-3 mx-auto"/>
+        <p style="color: #676A7A" class="mb-4">- Mike Arpaia, Co-creator of <a style="color:#676A7A;" target="_blank" href="https://osquery.io/">osquery</a></p>
+        <a target="_blank" href="https://osquery.io/"><img alt="osquery logo" src="/images/logo-osquery-89x24@2x.png" style="height: 40px; width: 137px;" class="d-inline-block mb-3 mx-auto"/></a>
```

Comment:
@mike-j-thomas I know it was already like this, but just fyi: I've had a lot of luck using this conventional ordering for HTML attributes.  @rachaelshaw and others have found it useful for styling: https://gist.github.com/mikermcneil/d3cad25d0cb9c225f749f5a2dac60ba2#html-attribute-order

Reply:
cc @gillespi314 @RachelElysia in case y'all want to get real nerdy about HTML with us (the tldr of the theory is morphology on the left, interactivity on the right.  That way when you're styling, you look left.  When you're wiring up interactions, you look right.)